ui/admin/app/templates/scopes/scope/session-recordings/index.hbs

```diff
@@ -15,118 +15,240 @@
     <p>{{t 'resources.session-recording.description'}}</p>
   </page.header>
 
-  <page.body>
-    {{#if this.hasSessionRecordings}}
-      <Hds::Table
-        @model={{@model.sessionRecordings}}
-        @density='tall'
-        @columns={{array
-          (hash label=(t 'resources.session-recording.form.time.label'))
-          (hash label=(t 'resources.session-recording.form.status.label'))
-          (hash label=(t 'resources.session-recording.form.user.label'))
-          (hash label=(t 'resources.session-recording.form.target.label'))
-          (hash label=(t 'resources.session-recording.form.duration.label'))
-          (hash label=(t 'actions.view') isVisuallyHidden=true align='right')
-        }}
-        @valign='middle'
-      >
-        <:body as |B|>
-          <B.Tr>
-            {{! Created Time }}
-            <B.Td>
-              {{#if B.data.created_time}}
-                <span class='hds-foreground-faint'>
-                  <time datetime={{format-date-iso B.data.created_time}}>
-                    {{format-date-iso B.data.created_time}}
-                  </time>
-                </span>
-              {{/if}}
-            </B.Td>
-            {{! Status }}
-            <B.Td>
-              <SessionRecording::Status @status={{B.data.state}} />
-            </B.Td>
-            {{! User }}
-            <B.Td>
-              <div
-                class='session-recording-list-cell-item hds-font-weight-semibold'
-              >
-                <span><FlightIcon @name='user' /></span>
-                <span>{{B.data.userDisplayName}}</span>
-              </div>
-            </B.Td>
-            {{! Target }}
-            <B.Td>
-              <div
-                class='session-recording-list-cell-item target-column hds-font-weight-semibold'
+  <page.body class='search-filtering'>
+    {{#if @model.sessionRecordingsExist}}
+      <div class='search-filtering-toolbar'>
+        <Hds::SegmentedGroup data-test-session-recordings-bar as |S|>
+          <S.TextInput
+            @value={{this.search}}
+            @type='search'
+            placeholder={{t 'actions.search-session-recording'}}
```

Comment:
non-blocking: just to clarify, is the reason why "Search recording ID" is the placeholder to let users know they can also search by id (since its not in the table) ?

Reply:
That is correct!

Reply:
Hmm is that needed? It makes me think it will only search by recording ID but it will search across all the fields you have indexed

Reply:
That is true. Thinking on this, you might be right that this will make the user think they can only search by id. I can change it to the normal search placeholder.